src/Illuminate/Encryption/Encrypter.php

```diff
@@ -70,7 +82,7 @@ public static function supported($key, $cipher)
      */
     public static function generateKey($cipher)
     {
-        return random_bytes($cipher === 'AES-128-CBC' ? 16 : 32);
```

Comment:
This is incorrect for AES-128-GCM, causing `key:generate` to output a 256 bit key and the application to throw a `RuntimeException`.